### PR TITLE
Front page calendar details dropdowns

### PR DESCRIPTION
**Pull request by juozasg**
Instead of mousenter/mouseleave I'm using clicks on the calendar items to activate. This works with both touchscreen and mouse cursor devices. Please review the import-calendar.py changes. I'm doing some data cleaning up that maybe could be avoided by changing calendar data contents, but it should work and look neat with the existing data now.

### CODE REVIEW

**Comment by dholbach** [comment]
Try changing the shebang to `python3` - maybe do that for all the scripts which don't have it yet.

I just used `netlify/build` to see what's available, it says:

```cli
buildbot@003a2a2e8c98:~/website$ python -V
Python 2.7.12
buildbot@003a2a2e8c98:~/website$
```

File: hack/import-calendar.py [hunk 1]
```diff
@@ -40,20 +62,47 @@ def download_calendar():
...
 
+
+def read_organizer(event):
+    organizer = event['organizer']
+    email = organizer.title().split(':')[1].lower()
+    name = email
+    if 'cn' in organizer.params:
+        name = organizer.params['cn']
+
+    mailto_href = ''.format(email, name)
```
Comment (dholbach) [inline hack/import-calendar.py:73]: Is this used anywhere?

Comment (juozasg) [reply hack/import-calendar.py:73]: It is not. Wil delete. Ty!

File: hack/import-calendar.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
-#!/usr/bin/python
+#!/usr/bin/python3.7
```
Comment (dholbach) [inline hack/import-calendar.py:1]:
```suggestion
#!/usr/bin/python3
```

This should be easier to fulfil on other setups, e.g. on Ubuntu 20.04:

```cli
❯ ls -l /usr/bin/python3
lrwxrwxrwx 1 root root 9 Mär 13  2020 /usr/bin/python3 -> python3.8
~ on ☁️  daniel@weave.works(europe-west1) 
❯ 
```

Comment (dholbach) [reply hack/import-calendar.py:1]: This might fix the link checker action as well.

Comment (juozasg) [reply hack/import-calendar.py:1]: I'm bit confused. I did this via a commit:

```
#!/usr/bin/python3
import sys
print(sys.version)
```
Commented out rest of the script. I got `3.5.7` from Netlify build log. Did you upgrade it elsewhere?

Comment (dholbach) [reply hack/import-calendar.py:1]: This is strange...... so maybe indeed we need to find a way to specifically point to 3.7 somehow. I thought that's what `runtime.txt` would do: https://docs.netlify.com/configure-builds/manage-dependencies/#python

Maybe let's set `PYTHON_VERSION` in the `Makefile` to be sure?

Comment (dholbach) [reply hack/import-calendar.py:1]: At some stage we should look into moving from Ubuntu 16.04 to Ubuntu 20.04 (just noticed that we're not on the newest yet). Will file a separate issue.

Comment (juozasg) [reply hack/import-calendar.py]: I set `PYTHON_VERSION := 3.7` in Makefile. Still getting `3.5` from `print(sys.version)`

File: Makefile [hunk 3]
```diff
@@ -23,6 +23,7 @@ HUGO_BIND_ADDRESS          ?= 127.0.0.1
 BUILDER_CLI                := docker
 # BUILDER_CLI                := okteto
 LYCHEE_IMAGE_NAME          ?= lycheeverse/lychee:202105190720247e4977
+PYTHON_VERSION						 := 3.7
```
Comment (dholbach) [inline Makefile:26]: This can be dropped now.

**Pull request merged**